src/gui/widgets/gt_shortcutedit.h

```diff
@@ -16,19 +16,40 @@
 
 #include "gt_lineedit.h"
 #include <QKeySequence>
+#include <QTableWidgetItem>
 
 /**
  * @brief The GtShortCutEdit class
  * Used to input a keysequence for a short cut (single key + multiple modifers).
  */
-class GT_GUI_EXPORT GtShortCutEdit : public GtLineEdit
+class GT_GUI_EXPORT GtShortCutEdit : public GtLineEdit, public QTableWidgetItem
 {
     Q_OBJECT
 
     Q_PROPERTY(QKeySequence keySequence READ keySequence WRITE setKeySequence)
 
 public:
 
+    /**
+     * @brief operator < for sorting GtLineEdits
+     * @param other
+     * @return
+     */
+    bool operator <(const QTableWidgetItem& other) const
+    {
+        if (other.column() == 1)
+        {
+            const GtLineEdit *p = dynamic_cast<const GtLineEdit *>(&other);
+
+            if (p)
+            {
+                if (this->GtLineEdit::text() < p->text()) return true;
+            }
```

Comment:
Please combine both if statements into one

```suggestion
            if (p && this->GtLineEdit::text() < p->text()) return true;
```